### PR TITLE
doc: Fix uses of not neutral pronouns

### PR DESCRIPTION
**Pull request by AuraHerreroRuiz**
There are some cases where the documentation or code comments use non-neutral pronouns. This PR changes these to use neutral pronouns to match the rest of the documentation.

### CODE REVIEW

**Review by ldv-alt — CHANGES_REQUESTED** [review]
I'm sorry but they/their is harder to read when applied to singular forms.  Fortunately, you can easily achieve the goal by tweaking these phrases, e.g. to use plural forms instead of singular.

File: doc/sag/Linux-PAM_SAG.xml [hunk 1]
```diff
@@ -132,8 +132,8 @@
     <para>
       To illustrate the flexibility you face, consider the following
       situation: a system administrator (parent) wishes to improve the
-      mathematical ability of her users (children). She can configure
-      their favorite ``Shoot 'em up game'' (PAM-aware of course) to
+      mathematical ability of their users (children). The system administrator
+      can configure their favorite ``Shoot 'em up game'' (PAM-aware of course) to
```
Comment (ldv-alt) [inline doc/sag/Linux-PAM_SAG.xml:136]: I suggest the following wording:
```
situation: system administrators (parents) wish to improve the
mathematical ability of their users (children). Their favorite
``Shoot 'em up game'' (PAM-aware of course) can be configured to
```

File: doc/specs/rfc86.0.txt [hunk 2]
```diff
@@ -402,7 +402,7 @@
 
 
 
-            given access to his/her account.  This function can implement
+            given access to their respective accounts.  This function can implement
```
Comment (ldv-alt) [inline doc/specs/rfc86.0.txt:405]: I suggest the following wording:

```
function to check whether the authenticated user should be
...
given access to the account.  This function can implement
```

File: libpam/pam_audit.c [hunk 3]
```diff
@@ -44,7 +44,7 @@ _pam_audit_writelog(pam_handle_t *pamh, int audit_fd, int type,
       free(buf);
   }
 
-  /* libaudit sets errno to his own negative error code. This can be
+  /* libaudit sets errno to the user's own negative error code. This can be
```
Comment (ldv-alt) [inline libpam/pam_audit.c:47]: The comment is talking about libaudit's own error code, so this edit changes the meaning.
I suggest the following wording:
`libaudit sets errno to its own negative error code.`

File: modules/pam_env/pam_env.conf.5.xml [hunk 4]
```diff
@@ -58,7 +58,7 @@
       For example, ${HOME} is used below several times, but
       many PAM applications don't make it available by the time you need it.
       The special variables @{HOME} and @{SHELL} are expanded to the values
-      for the user from his <emphasis>passwd</emphasis> entry.
+      for the users' respective <emphasis>passwd</emphasis> entries.
```
Comment (ldv-alt) [inline modules/pam_env/pam_env.conf.5.xml:61]: I suggest the following wording:
`for the user from the corresponding <emphasis>passwd</emphasis> entry.`

File: modules/pam_timestamp/pam_timestamp_check.8.xml [hunk 5]
```diff
@@ -72,7 +72,7 @@ see if the default timestamp is valid, or optionally remove it.
             <para>
                By default <command>pam_timestamp_check</command> checks or removes
                timestamps generated by <emphasis>pam_timestamp</emphasis> when
-               the user authenticates as herself. When the user authenticates as a
+               users authenticate as themselves. When users authenticate as a
```
Comment (ldv-alt) [inline modules/pam_timestamp/pam_timestamp_check.8.xml:75]: I suggest the following wording:
`users authenticate as themselves. When the user authenticates as a`

**Review by ldv-alt — CHANGES_REQUESTED** [review]

**Comment by AuraHerreroRuiz** [comment]
I've committed your suggestions, let me know if you need further changes

**Review by ldv-alt — APPROVED** [review]
Looks fine now.  I've rebased and squashed the commits.

**Pull request merged**